drivers/npmx/npmx_shell.c

```diff
@@ -1667,20 +1843,56 @@ SHELL_STATIC_SUBCMD_SET_CREATE(sub_charger_trickle,
 					 NULL),
 			       SHELL_SUBCMD_SET_END);
 
-/* Creating subcommands (level 2 command) array for command "charger". */
-SHELL_STATIC_SUBCMD_SET_CREATE(sub_charger,
-			       SHELL_CMD(termination_voltage, &sub_charger_termination_voltage,
-					 "Charger termination voltage", NULL),
-			       SHELL_CMD(termination_current, &sub_charger_termination_current,
-					 "Charger termination current", NULL),
-			       SHELL_CMD(charger_current, &sub_charger_charging_current,
-					 "Charger current", NULL),
-			       SHELL_CMD(status, &sub_charger_status, "Charger status", NULL),
-			       SHELL_CMD(module, &sub_charger_module, "Charger module", NULL),
-			       SHELL_CMD(trickle, &sub_charger_trickle, "Charger trickle voltage",
-					 NULL),
+/* Creating subcommands (level 4 command) array for command "charger die_temp status". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_die_temp_status,
+			       SHELL_CMD(get, NULL,
+					 "Get die temperature comparator status",
+					 cmd_die_temp_status_get),
 			       SHELL_SUBCMD_SET_END);
 
+/* Creating subcommands (level 4 command) array for command "charger die_temp stop". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_die_temp_stop,
+			       SHELL_CMD(get, NULL,
+					 "Get die temperature threshold for stop charging",
+					 cmd_die_temp_stop_get),
+			       SHELL_CMD(set, NULL,
+					 "Set die temperature threshold for stop charging",
+					 cmd_die_temp_stop_set),
+			       SHELL_SUBCMD_SET_END);
+
+/* Creating subcommands (level 4 command) array for command "charger die_temp resume". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_die_temp_resume,
+			       SHELL_CMD(get, NULL,
+					 "Get die temperature threshold for resuming charging",
+					 cmd_die_temp_resume_get),
+			       SHELL_CMD(set, NULL,
+					 "Set die temperature threshold for resuming charging",
+					 cmd_die_temp_resume_set),
+			       SHELL_SUBCMD_SET_END);
+
+/* Creating subcommands (level 3 command) array for command "charger die_temp". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_die_temp,
+			       SHELL_CMD(resume, &sub_die_temp_resume,
+					 "Die temperature threshold where charging resumes", NULL),
+			       SHELL_CMD(stop, &sub_die_temp_stop,
+					 "Die temperature threshold where charging stops", NULL),
+					SHELL_CMD(status, &sub_die_temp_status,
+					 "Die temperature threshold where charging stops", NULL),
```

Comment:
```suggestion
			       SHELL_CMD(stop, &sub_die_temp_stop,
					 "Die temperature threshold where charging stops", NULL),
				SHELL_CMD(status, &sub_die_temp_status,
					 "Die temperature threshold where charging stops", NULL),
```
Looks like lack of indentation.